pepys_import/database/default_data/Nationalities.csv

```diff
@@ -254,4 +254,5 @@ Wallis and Futuna,
 Western Sahara,
 Yemen,
 Zambia,
-Zimbabwe,
\ No newline at end of file
+Zimbabwe,
+Unknown,
```

Comment:
@robintw - we won't have these 3 `Unknown` values in the analyst's PostGis instance.  Does Pepys generate the value if it isn't already preasent?  Or do we need to run a migration?

(yes - this question will probably get answered as I get further down the review 🤣  )

Reply:
It didn't, but it does now.